docs/usage/cli/filter.rst

```diff
@@ -63,3 +71,184 @@ The filter command allows you to select a specific number of sequences from spec
+filter``. Additionally, there is an extra step to combine the intermediate
+samples.
+
+    1. Sample 20 sequences from North America.
+    2. Sample 50 sequences from Europe.
+    3. Combine the samples.
+
+Calling ``augur filter`` multiple times
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create each intermediate sample using ``augur filter``, then join them together
+to create the final subsample.
+
+.. code-block:: bash
+
+    # 1. Sample 20 sequences from North America
+	augur filter \
+      --sequences sequences.fasta \
+      --metadata metadata.tsv \
+      --query "region == 'North America'" \
+      --subsample-max-sequences 20 \
+      --output-sequences north_america_sample.fasta \
+      --output-metadata north_america_sample.tsv
+
+    # 2. Sample 50 sequences from Europe
+	augur filter \
+      --sequences sequences.fasta \
+      --metadata metadata.tsv \
+      --query "region == 'Europe'" \
+      --subsample-max-sequences 50 \
+      --output-sequences europe_sample.fasta \
+      --output-metadata europe_sample.tsv
+
+    # 3.1 Combine FASTA files
+    cat north_america_sample.fasta europe_sample.fasta > combined.fasta
+
+    # 3.2 Combine metadata files, using tail to skip the extra header row
+    cat north_america_sample.tsv > combined.tsv
+    tail -n+2 europe_sample.tsv >> combined.tsv
+
+The above approach works fine, but if no sequence filters are used, there is a better approach:
+
+.. code-block:: bash
+
+    # 1. Sample 20 sequences from North America
+	augur filter \
+      --metadata metadata.tsv \
+      --query "region == 'North America'" \
+      --subsample-max-sequences 20 \
+      --output-strains north_america_sample_strains.txt
+
+    # 2. Sample 50 sequences from Europe
+	augur filter \
+      --metadata metadata.tsv \
+      --query "region == 'Europe'" \
+      --subsample-max-sequences 50 \
+      --output-metadata europe_sample_strains.txt
+
+    # 3. Combine using augur filter
+	augur filter \
+      --sequences sequences.fasta \
+      --metadata metadata.tsv \
+      --exclude-all \
+      --include north_america_sample_strains.txt \
+                europe_sample_strains.txt \
+      --output-sequences combined.fasta \
+      --output-metadata combined.tsv
+
+Differences:
+
+- In the first two steps, ``--input-sequences`` is removed.
+- In the first two steps, ``--output-sequences`` and ``--output-metadata`` are
+  replaced by ``--output-strains``.
+- The third step of combining the two intermediate samples is done using ``augur
+  filter`` itself. This uses ``augur filter`` to sample the data based on an
+  exact list of strains using a combination of ``--exclude-all`` and ``--include``.
+
+For large datasets, this has a computational benefit. Reading through a large
+FASTA file can take a long time. Instead of reading through it twice in the
+first two steps, read through it once in the final step.
+
+Stitching calls together with Snakemake
```

Comment:
I might rename this section to replace "Stitching calls together with Snakemake" with "Generalizing subsampling in a workflow" or "Automating subsampling in a workflow", since the concept isn’t Snakemake-specific even though the implementation example is.

Reply:
Updated to use "Generalizing subsampling in a workflow" and tweaked the first paragraph: 5e6569a998aa40a77386b0da559908114d49e19c